src/index.js

```diff
@@ -1,3 +1,8 @@
+// Add polyfill for `fetch`
+require('whatwg-fetch');
```

Comment:
Shouldn't this be also added to `package.json`?

Reply:
This is already in the package.json because we use it for the public bundle.

Reply:
:+1: